examples/Tutorial_cellpack.ipynb

```diff
@@ -0,0 +1,164 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Simularium Conversion Tutorial : Cellpack Data\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from IPython.display import Image\n",
+    "from simulariumio.cellpack import CellpackConverter, CellpackData\n",
+    "from simulariumio import (\n",
+    "    UnitData,\n",
+    "    MetaData,\n",
+    "    DISPLAY_TYPE,\n",
+    "    ModelMetaData,\n",
+    "    InputFileData,\n",
+    ")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook provides example python code for converting your own simulation trajectories into the format consumed by the Simularium Viewer. It creates a .simularium JSON file which you can drag and drop onto the viewer like this:\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![title](img/drag_drop.gif)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "\n",
+    "## Prepare your spatial data\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Simularium `CellpackConverter` consumes spatiotemporal data from Cellpack.\n",
+    "\n",
+    "The converter requires a `CellpackData` object as a parameter ([see documentation](https://allen-cell-animated.github.io/simulariumio/simulariumio.cellpack.html#simulariumio.cellpack.cellpack_data.CellpackData)).\n",
+    "\n",
+    "If you'd like to specify PDB or OBJ files or color for rendering an agent type, add a `DisplayData` object for that agent type, as shown below ([see documentation](https://allen-cell-animated.github.io/simulariumio/simulariumio.data_objects.html#module-simulariumio.data_objects.display_data)).\n"
```

Comment:
Are these allen-cell-animated URLs still valid?

Reply:
no, but I think they exist in the other tutorials as well... I just updated this file for now, and I'll make an issue.